traffic_ops/traffic_ops_golang/dbhelpers/db_helpers.go

```diff
@@ -897,7 +895,7 @@ func DSRExistsWithXMLID(xmlid string, tx *sql.Tx) (bool, error) {
 	return exists, err
 }
 
-// ScanCachegroupsServerCapabilities, given rows of (server ID, CDN ID, cachegroup name, server capabilities),
+// ScanCachegroupsServerCapabilities : given rows of (server ID, CDN ID, cachegroup name, server capabilities),
```

Comment:
this was grammatically correct before, the colon should be used to separate independent clauses, but neither side of this colon is an independent clause.